code/modules/power/apc.dm

```diff
@@ -633,7 +633,7 @@
 			update_icon()
 		return
 	else if(istype(W, /obj/item/apc_powercord))
-		return//because we put our fancy code in the right places, and this is all in the powercord's afterattack() >:C
+		return
```

Comment:
Please leave the comment there. Can be less inventive, but the situation should be explained.